Ginger/GingerCoreNET/ActionsLib/ActScript.cs

```diff
@@ -198,6 +200,11 @@ public override void Execute()
                         p.StartInfo.FileName = System.IO.Path.Combine(p.StartInfo.WorkingDirectory, ScriptName);
                         p.StartInfo.Arguments = Params;
                     }
+                    else if (ScriptInterpreterType == eScriptInterpreterType.SH)
+                    {
+                        string filePath = amdocs.ginger.GingerCoreNET.WorkSpace.Instance.SolutionRepository.ConvertSolutionRelativePath(Path.Combine(SolutionFolder, @"Documents/scripts/") + ScriptName);
```

Comment:
why to mention again "Documents/script" if you handled it in p.StartInfo.WorkingDirectory?

Reply:
fixed